### ISSUE TITLE
IE11 does not display content - compatibility issues with CSS Flex

### ISSUE THREAD

**Issue by shaunanoordin** [zooniverse/shakespeares_world#145]
Expected Behavour:
* Visiting the app should actually display some content - e.g. the "home page" with the "Shakespeare's World" title and the "Get Started" button.

Actual Behaviour
* Visiting the app on IE11 shows no content - only the project header and the Zooniverse footer.

![zooniverse-20151106-folgerdemo-ie11](https://cloud.githubusercontent.com/assets/13952701/11004270/2c9d4224-84af-11e5-84c4-fe3041470249.png)

Diagnosis:
* IE11 does not like the CSS flex functionality.
* Point of error appears to be in the main content element - ".layout-container > .main" in main.css - which uses flex statements.

Suggested Solution:
* Examine possible duplicate issue on Old Weather - https://github.com/zooniverse/old-weather/pull/76

**Comment by simoneduca** [comment]
Try this http://philipwalton.com/articles/normalizing-cross-browser-flexbox-bugs/

**Comment by shaunanoordin** [comment]
This looks like a promising fix for IE11:

```
body > div.ng-scope {
    display: flex;
    flex-direction: column;
}
.layout-container > .main {
    flex: 1 0 auto;
}
```

Analysis:
* First issue identified is the min-height bug in IE11, which is detailed in https://github.com/philipwalton/flexbugs item 3.
  * Instead of setting .layout-container to height:100vh, we set .layout-container's parent to display: flex, flex-direction: column.
* Second issue identified is that in IE11, the main container is constantly shrunk to accommodate space for the footer in the viewport. (i.e. footer was sticky to the bottom of the window, not the bottom of the document content.)
  * This issue can be rectified by telling the main container to stop shrinking; ```flex: 1 1 auto -> flex: 1 0 auto```

Tested On:
* IE11, Chrome 46, FF 42, Opera 33 & Edge 20 / Win 10 - layout looks stable on localhost replication with above CSS fixes.
* NOT tested on Safari.
